Packages/com.unity.inputsystem/InputSystem/Devices/Pointer.cs

```diff
@@ -57,20 +56,16 @@ internal struct PointerState : IInputStateTypeInfo
         [InputControl(layout = "Analog", usage = "Pressure")]
         public float pressure;
 
-        [InputControl(layout = "Axis", usage = "Twist")]
-        public float twist;
-
-        [InputControl(layout = "Vector2", usage = "Tilt")]
-        public Vector2 tilt;
+        float twist;
+        Vector2 tilt;
 
         [InputControl(layout = "Vector2", usage = "Radius")]
         public Vector2 radius;
 
         [InputControl(name = "press", layout = "Button", format = "BIT", bit = 0)]
         public ushort buttons;
 
-        [InputControl(layout = "Digital")]
-        public ushort displayIndex;
+        ushort displayIndex;
```

Comment:
This can be removed.

Reply:
On `PenState` and `MouseState`, the `[InputControl...]` annotations for `displayIndex` need to be removed. Fields can be made internal (I guess removing them breaks il2cpp). ATM both `PenState` and `MouseState` will separately re-introduce `displayIndex`.

Reply:
Nope, removing this will break il2cpp struct layout. (Brought it back after I removed it first and stuff stopped working on my phone).

Reply:
But will remove the [InputControl...] annotations  i missed

Reply:
Would prefer a comment on these fields why they are there. Probably also makes sense to just rename all the leftover fields we are keeping for the sake of il2cpp to something like `_paddingXXX` or so.